readthedocs/search/tests/test_views.py

```diff
@@ -18,7 +18,10 @@
 @pytest.mark.django_db
 @pytest.mark.search
 class TestProjectSearch:
-    url = reverse('search')
+
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        self.url =  reverse('search')
```

Comment:
Had to change this, because pytest evaluates/import the file, in proxito we don't have `search` (yet)